### ISSUE TITLE
How to evaluate the fine-tuned model using a custom dataset?

### ISSUE THREAD

**Issue by AllenReder** [bytedance/Sa2VA#106]
Thank you for your outstanding work!

I have fine-tuned the model with my own dataset, but I'm not quite sure how to evaluate the validation set or test set. Do I need to copy and modify `projects/sa2va/evaluation/sa2va_eval_refcoco.py` to implement the new evaluation code, or have I missed something?

**Comment by HarborYuan** [comment]
Yes. I think so. Before that, you may need to convert your model to HF format for evaluation.

> tools/convert_to_hf.py

**Comment by HarborYuan** [comment]
I think the easiest way is to write a custom dataset with each item as the following format:

{
  "img_id": xx,
  "image": xx,
  "prompt": xx,
}

and then write some logics to parse the dataset to feed into the Sa2VA model.

**Comment by YachaoDong** [comment]
> 感谢你们出色的工作！
> 
> 我已经用自己的数据集对模型进行了微调，但我不太确定如何评估验证集或测试集。我需要复制并修改代码`projects/sa2va/evaluation/sa2va_eval_refcoco.py`来实现新的评估功能吗？还是我遗漏了什么？

请问微调自己的数据集效果咋样哇？和实例分割小模型相比呢？
